view/frontend/layout/catalog_product_view_type_grouped.xml

```diff
@@ -3,6 +3,7 @@
     <body>
 
         <referenceBlock name="breadcheckout.prod.view" remove="true"/>
+        <referenceBlock name="breadcheckout.prod.price" remove="true"/>
```

Comment:
Should this option be disabled for grouped products, or this is something that will be added later on?